.circleci/config.yml

```diff
@@ -2,7 +2,7 @@ version: 2.0
 jobs:
   build:
     docker:
-      - image: docker.io/champtar/openwrtpackagesci@sha256:96ef72edc70cba371ea5676fba15ee25b3a94f538f648a27454b699edce61da0
+      - image: docker.io/openwrtorg/packages-cci:v1.0.1
```

Comment:
I prefer using digest instead of tags so nobody can change it without a proper commit in the repo

Reply:
Groan - I did consider using the SHA digest but to me, besides being somewhat meaningless, there is no sense of history in Docker Hub repos. I guess I'm not paranoid enough about people with write access to the repo not keeping some sort of versioning scheme. Any missteps can always be corrected.

Do you feel strongly about such matters? Do we need some hybrid?

Reply:
It's not about security, more about least surprise, you are sure of what ran, but I can totally live with tags